pkg/scheduler/eventhandlers.go

```diff
@@ -35,12 +30,17 @@ import (
 	"k8s.io/client-go/tools/cache"
 	"k8s.io/klog/v2"
 
+	nodev1alpha1 "github.com/kubewharf/godel-scheduler-api/pkg/apis/node/v1alpha1"
```

Comment:
Should packages from other repos (godel-scheduler-api, katalyst-api) be located in above?